package.json

```diff
@@ -3,8 +3,13 @@
   "version": "2.1.0",
   "description": "k-neareset neighbors search for RBush",
   "main": "index.js",
+  "jsdelivr": "rbush-knn.js",
+  "unpkg": "rbush-knn.js",
```

Comment:
btw I think we should target `min` versions in CDN defaults

Reply:
agreed, done.